samples/exposed-migration/README.md

```diff
@@ -0,0 +1,3 @@
+# Exposed Migration
+
+This sample project uses Flyway to perform a migration with a migration script generated by Exposed.
```

Comment:
Could you please give some details what this project illustrate how to read it and run